documentation/specs/src/interoperability/ethereum-bridge/bootstrapping.md

```diff
@@ -1,4 +1,73 @@
 # Bootstrapping the bridge
 
-Before the bridge can start running, some storage may need to be initialized in 
-Namada. TBD.
\ No newline at end of file
+## Overview
+
+The Ethereum bridge is not enabled at the launch of a Namada chain. Instead, there is a governance parameter, `eth_bridge_proxy_address`, which is initialized to the zero Ethereum address (`"0x0000000000000000000000000000000000000000"`). An overview of the steps to enable the Ethereum bridge for a given Namada chain are:
+
+- A governance proposal should be held to agree on a block height `h` at which to launch the Ethereum bridge by means of a hard fork.
+- If the proposal passes, the Namada chain must halt after finalizing block `h-1`. This requires
+- The [Ethereum bridge smart contracts](./ethereum_smart_contracts.md) are deployed to the relevant EVM chain, with the active validator set at block height `h` as the initial validator set that controls the bridge.
+- Details are published so that the deployed contracts can be verified by anyone who wishes to do so.
+- If active validators for block height `h` regard the deployment as valid, the chain should be restarted with a new genesis file that specifies `eth_bridge_proxy_address` as the Ethereum address of the proxy contract.
+
+At this point, the bridge is launched and it may start being used. Validators' ledger nodes will immediately and automatically coordinate in order to craft the first validator set update protocol transaction.
+
+## Facets
+
+### Governance proposal
+
+The governance proposal can be freeform and simply indicate what the value of `h` should be. Validators should then configure their nodes to halt at this height. The `grace_epoch` is arbitrary as there is no code to be executed as part of the proposal, instead validators must take action manually as soon as the proposal passes. `h` must be in an epoch that is strictly greater than `voting_end_epoch`.
```

Comment:
I know this is a super tiny nit, but sentences not beginning with a capital letter bother me.
```suggestion
The governance proposal can be freeform and simply indicate what the value of `h` should be. Validators should then configure their nodes to halt at this height. The `grace_epoch` is arbitrary as there is no code to be executed as part of the proposal, instead validators must take action manually as soon as the proposal passes. The block height `h` must be in an epoch that is strictly greater than `voting_end_epoch`.
```